dota-enums.d.ts

```diff
@@ -1843,7 +1843,7 @@ declare enum modifierfunction {
      */
     MODIFIER_PROPERTY_BONUSDAMAGEOUTGOING_PERCENTAGE,
     /**
-     * Unknow, GetAttackWhileMovingTarget?
+     * Unknow, GetModifierAttackWhileMovingTarget?
```

Comment:
Is this still unknown or is it actually GetModifierAttackWhileMovingTarget? IN that case you can just remove unknow (or fix it to `unknown`)

Reply:
OK, I think it's better to delete. There is no function name of this constant in script_help2.

Reply:
Could you then just keep it like it is? If the enum exists it should be in declarations.

Reply:
Ok, I keep it.